src/writer/creator.cpp

```diff
@@ -206,15 +206,15 @@ namespace zim
     {
       // Create mandatory entries
       if (!m_faviconPath.empty()) {
-        auto dirent = data->createRedirectDirent('-', "favicon", "", 'C', m_faviconPath);
+        auto dirent = data->createRedirectDirent('W', "favicon", "", 'C', m_faviconPath);
```

Comment:
Why can't these small changes related to well-known entries go into a separate small PR?

Reply:
Moved out the PR.